pytorch_lightning/core/saving.py

```diff
@@ -348,11 +349,14 @@ def load_hparams_from_yaml(config_yaml: str, use_omegaconf: bool = True) -> Dict
     return hparams
 
 
-def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
+def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace], use_omegaconf: bool = True) -> None:
     """
     Args:
         config_yaml: path to new YAML file
         hparams: parameters to be saved
+        use_omegaconf: If both `OMEGACONF_AVAILABLE` and `use_omegaconf` are True,
+                the hparams will be converted to `DictConfig` if possible
+
```

Comment:
```suggestion
        use_omegaconf: If both omegaconf is available, the hparams will be converted to ``DictConfig`` if possible
```
the reader of the docstring has no insight into the local variable OMEGACONF_AVAILABLE